e2e/nodedrain/nodedrain.go

```diff
@@ -245,6 +245,7 @@ func (tc *NodeDrainE2ETest) TestNodeDrainIgnoreSystem(f *framework.F) {
 // TestNodeDrainDeadline tests the enforcement of the node drain deadline so
 // that allocations are terminated even if they haven't gracefully exited.
 func (tc *NodeDrainE2ETest) TestNodeDrainDeadline(f *framework.F) {
+	t.Skip("The behavior is unclear and test assertions don't capture intent.  Issue 9902")
```

Comment:
I think these need to be `f.T().Skip`?

Reply:
Yeah, caught it when linter failed as well. Not sure why my editor didn't report it.